README.md

```diff
@@ -1,2 +1,126 @@
-# Awesome-Books
-A book list app(Microverse Project)
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Awesome Books App <a name="about-project"></a>
+
+
+**Awesome Books App** is an app that allows you to add and delete books and it stores them for you in object arrays
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- Javascript
+
+
+<!-- Features -->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- Link is not available yet will add it soon
+
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+clone this ssh key git@github.com:Munyabelden/Awesome-Books.git using vs code or git bash
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Git and Github
+- Vs code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+ssh key git@github.com:Munyabelden/Awesome-Books.git
+
+
+
+
+### Deployment
+
+You can deploy this project using:
+
+- github page or any other hosting you like
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Belden Mugauri**
+
+- GitHub: [@githubhandle](https://github.com/Munyabelden)
+- Twitter: [@twitterhandle](https://twitter.com/home)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/feed/)
+
+👤 **Tesfaye Tecklu**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Munyabelden/Awesome-Books/issues).
+
```

Comment:
- [ ] As requested by the previous reviewer [here](https://github.com/Munyabelden/Awesome-Books/pull/1#discussion_r1112100257) kindly add the acknowledgment section to your README.